library/src/main/java/com/fidesmo/fdsm/FidesmoApiClient.java

```diff
@@ -193,6 +193,7 @@ public JsonNode rpc(URI uri, JsonNode request) throws IOException {
 
         req.setHeader(HttpHeaders.ACCEPT, ContentType.APPLICATION_JSON.toString());
         req.setHeader(HttpHeaders.CONTENT_TYPE, ContentType.APPLICATION_JSON.toString());
+        req.setHeader("Protocol-Requirements", "parametrisedTranslations");
```

Comment:
Isn't that a header that is set internally, while externally it should be just API v3? @jjluczyn-fidesmo can you help?
But what we can do is send the Accept Language to the server in every request (if not yet)